From 62cccd99e42d16614ed1ebdbc1ed888436187f29 Mon Sep 17 00:00:00 2001
From: aprantl <aprantl@coremedia.com>
Date: Tue, 1 Mar 2022 11:17:24 +0100
Subject: [PATCH] ci/develop Only: Set studio-client versions and relativePath

Set version and relativePath, so that the workspace can be used
in CoreMedia CI Systems.

diff --git a/apps/studio-server/pom.xml b/apps/studio-server/pom.xml
index 69340a151082..546b459447db 100644
--- a/apps/studio-server/pom.xml
+++ b/apps/studio-server/pom.xml
@@ -5,6 +5,7 @@
     <artifactId>studio-server.extensions</artifactId>
     <groupId>com.coremedia.blueprint</groupId>
     <version>1-SNAPSHOT</version>
+    <relativePath>../../../../../../apps/studio-server/blueprint/modules/extensions/pom.xml</relativePath>
   </parent>
 
     <groupId>com.coremedia.labs.translation.gcc</groupId>
diff --git a/apps/user-changes/pom.xml b/apps/user-changes/pom.xml
index fa15ad056a52..fe25611896c4 100644
--- a/apps/user-changes/pom.xml
+++ b/apps/user-changes/pom.xml
@@ -5,6 +5,7 @@
     <artifactId>user-changes.extensions</artifactId>
     <groupId>com.coremedia.blueprint</groupId>
     <version>1-SNAPSHOT</version>
+    <relativePath>../../../../../../apps/user-changes/blueprint/modules/extensions/pom.xml</relativePath>
   </parent>
 
     <groupId>com.coremedia.labs.translation.gcc</groupId>
diff --git a/apps/workflow-server/pom.xml b/apps/workflow-server/pom.xml
index 20a336499db4..d5c27b7c0da4 100644
--- a/apps/workflow-server/pom.xml
+++ b/apps/workflow-server/pom.xml
@@ -6,6 +6,7 @@
     <artifactId>workflow-server.extensions</artifactId>
     <groupId>com.coremedia.blueprint</groupId>
     <version>1-SNAPSHOT</version>
+    <relativePath>../../../../../../apps/workflow-server/blueprint/modules/extensions/pom.xml</relativePath>
   </parent>
 
   <packaging>pom</packaging>
diff --git a/content/pom.xml b/content/pom.xml
index b8df248eaaaa..273d54f8c689 100644
--- a/content/pom.xml
+++ b/content/pom.xml
@@ -4,6 +4,7 @@
     <groupId>com.coremedia.blueprint</groupId>
     <artifactId>content.extensions</artifactId>
     <version>1-SNAPSHOT</version>
+    <relativePath>../../../../../content/blueprint/modules/extensions/pom.xml</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
